src/main/java/dook/tasks/Task.java

```diff
@@ -0,0 +1,37 @@
+package dook.tasks;
+
+public class Task {
+    protected String description;
+    protected Boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
```

Comment:
I like that even though these are public methods, since they are getters / setters, you omitted the JavaDoc, as it says in the coding standard that you may omit Javadocs for getters and setters.